page-objects/sentence-plan-pages.ts

```diff
@@ -368,12 +368,114 @@ export class SentencePlanPage {
         for (const locator of missingInfoSections) {
             await expect(locator).toBeVisible();
         }
+        const allTextMissingInfoSections = await missingInfoAccordion.allTextContents();
+        const sortedMissingInfoSections = [...allTextMissingInfoSections].sort((a, b) => a.localeCompare(b));
+        // Assert sections are in alaphabetical order
+        expect(allTextMissingInfoSections).toEqual(sortedMissingInfoSections);
     }
 
     async checkNoInfoAvailableDisplays() {
         await newTabGlobal!.locator('#accommodation > div.govuk-accordion__section-header > h3 > button').click();
         await expect(newTabGlobal!.getByText('No information is available yet'))
             .toBeVisible();
+    }
 
+    async checkInfoSectionNoFlagsListsCorrectOrder() {
+        // High scoring
+        const highScoringInfoAccordion = newTabGlobal!.locator('#assessment-accordion-highScoring')
+        await expect(highScoringInfoAccordion).toBeVisible();
+        await expect(highScoringInfoAccordion).toContainText('Accommodation');
+        await expect(highScoringInfoAccordion).toContainText('Personal relationships and community');
+        await expect(highScoringInfoAccordion).toContainText('Thinking, behaviours and attitudes');
+        await expect(highScoringInfoAccordion).toContainText('Drug use');
+        await expect(highScoringInfoAccordion).toContainText('Employment and education');
+        const highScoringInfoSections = [
+            newTabGlobal!.getByLabel('Accommodation'),
+            newTabGlobal!.getByLabel('Personal relationships and'),
+            newTabGlobal!.getByLabel('Thinking, behaviours and'),
+            newTabGlobal!.getByLabel('Drug use'),
+            newTabGlobal!.getByLabel('Employment and education')
+        ];
+        for (const locator of highScoringInfoSections) {
+            await expect(locator).toBeVisible();
+            const positions = await locator.evaluateAll(items => items.map(item => item.getBoundingClientRect().y));
+            const sortedPositions = [...positions].sort((a, b) => a - b);
```

Comment:
not sure I understand this - it gets the y position of each assessment area in this section, then compares the order of that list to the sorted value - in which case wouldn't an alphabetical order do the same thing?

Reply:
So here I had to go for something dynamic as the order is not alphabetical, so I wanted indeed to get the y position before matching than against the sorted value. But then looking at things again I thought it would be more uniform to use this way for the alphabetical order in the scenario involving the missing information sections. I've pushed the change, hope it's clearer!